.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 - [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
 - [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
-- [ ] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
+- [ ] Have you built your formula locally prior to submission with `brew install --build-from-source <formula>` (where `<formula>` is the name of the formula you're submitting)?
```

Comment:
Good point from @ilovezfs: let's go for `brew uninstall --force foo`, `brew install --build-from-source foo`